about_binaries.exs

```diff
@@ -31,4 +31,46 @@ defmodule AboutBinaries do
     contatenated = "Hello " <> "world"
     assert contatenated == __?
   end
+
+  def bit_representation(integer) do
+    Integer.to_string(integer, 2) # base 2, AKA "binary"
+  end
+
+  @tag :focus
+  think "Numbers can be represented in base 2 (binary) as a sequence of bits" do
+    assert bit_representation(1) == "1"
+    assert bit_representation(2) == "10"
+    assert bit_representation(3) == "11"
+    assert bit_representation(4) == "100"
+  end
+
+  @tag :focus
+  think "255 is the biggest value we can represent with 8 bits (1 byte)" do
+    assert bit_representation(255) == "11111111"
+    assert String.length(bit_representation(255)) == 8
+    assert bit_representation(256) == "100000000"
+    assert String.length(bit_representation(256)) == 9
+  end
+
+  @tag :focus
+  think "Unicode defines far more than 255 codepoints" do
+    assert ?λ == 955
+    assert ?™ == 8482
+  end
+
+  @tag :focus
+  think "Therefore, most Unicode codepoints must be encoded using more than 1 byte" do
+    assert ?λ == 955
+    assert String.length("λ") == 1
+    assert byte_size("λ") == 2
+    assert "λ" === <<206, 187>>
```

Comment:
I definitely don't have a good understanding of unicode. What's the significance of 206 and 187? I might have thought i'd take more than 2 bytes since 955 > 256*2.